### DISCUSSION TITLE
Website Improvement

### DISCUSSION THREAD

**Discussion by TheSCREWEDSoftware** [chromiecraft/chromiecraft#7363] [Suggestions]
Related to the donations/rewards-catalogue part of the website.

Increase the row by 1 or 2 (1 maybe looks the better) would spread the current 10 elemnts in to a more readable method as screen tend to be wider. 
(Example below)
1 2 3 4 | 1 2 3 4 5
5 6 7 8 | 6 7 8 9 10
9 10 

The interaction of the text should be extended to the icons itself. Still keeping the hover for the value, but allowing the clicking of the images to match the text clicabiltiy.

**Comment by Yehonal** [comment]
This are valid suggestions and we were already aware of the "icon issue"

We are currently very busy but we will work on it as soon as we will have the capacity. 

Thanks for reporting!

**Comment by Metal-Luna** [comment]
possible solution provided in https://github.com/chromiecraft/chromiecraft/issues/9487
